ocs_sample_library_preview/DataView/DataView.py

```diff
@@ -1,367 +1,354 @@
 import json
+
 from ..SDS.SdsTypeCode import SdsTypeCode
-from .Query import Query
-from .FieldSet import FieldSet
+from .DataViewShape import DataViewShape
 from .Field import Field
-from .DataViewShapes import DataViewShapes
+from .FieldSet import FieldSet
+from .Query import Query
 
 
 class DataView(object):
-    """
-    Data View definition
-    """
-
-    def __init__(
-        self,
-        id=None,
-        name=None,
-        description=None,
-        queries=None,
-        indexField=None,
-        dataFieldSets=None,
-        groupingFields=None,
-        indexTypeCode=None,
-        defaultStartIndex=None,
-        defaultEndIndex=None,
-        defaultInterval=None,
-        shape=None
-    ):
+    """OCS Data View definition"""
+
+    def __init__(self, id: str = None, name: str = None, description: str = None,
+                 index_field: Field = None, queries: list[Query] = None,
```

Comment:
does it matter that queries and index_type_code were reordered a bit? if someone were using unnamed arguments to initialize this object, they would be mapping to different parameters now

Reply:
Not necessarily, but since I don't think there was really any good reason for the reordering I reverted that change.